wallet/src/main/java/com/android/identity_credential/wallet/ui/destination/addtowallet/AddToWalletScreen.kt

```diff
@@ -218,4 +285,46 @@ private fun AddToWalletScreenWithIssuerDisplayDatas(
             }
         }
     }
+
+    // Scan Credential Offer
+    FilledTonalButton(
+        modifier = Modifier
+            .fillMaxWidth()
+            .padding(8.dp),
+
+        colors = ButtonColors(
+            containerColor = MaterialTheme.colorScheme.primaryContainer,
+            contentColor = MaterialTheme.colorScheme.onPrimaryContainer,
+            disabledContainerColor = Color.Green,
+            disabledContentColor = Color.Green
+        ),
+
+        shape = RoundedCornerShape(8.dp),
+        onClick = {
+            onShowScanQrDialog.invoke()
+        }) {
+        Row {
+            Image(
+                painter = painterResource(id = R.drawable.qr_icon),
+                contentDescription = stringResource(R.string.accessibility_artwork_for)
```

Comment:
This doesn't look right. String append won't do the right substitution.

Reply:
Good eye!